### PR TITLE
[Method References] LPS-162384 Convert method references to field references in modules commerce-punchout、portal*、segments and vldap

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang , I convert method references to field references in modules commerce-punchout、portal*、segments and vldap.

### CODE REVIEW

**Comment by W2jx** [comment]
Hi @zhangyan0701 , Could you please help me check the test results when they finish ? Thanks!

File: modules/dxp/apps/portal-reports-engine-console/portal-reports-engine-console-web/src/main/java/com/liferay/portal/reports/engine/console/web/internal/admin/portlet/AdminPortlet.java [hunk 1]
```diff
@@ -159,6 +147,12 @@ protected void activate(Map<String, Object> properties) {
 				ReportsEngineAdminWebConfiguration.class, properties);
 	}
 
+	@Reference
+	protected DefinitionLocalService definitionLocalService;
```
Comment (dantewang) [inline modules/dxp/apps/portal-reports-engine-console/portal-reports-engine-console-web/src/main/java/com/liferay/portal/reports/engine/console/web/internal/admin/portlet/AdminPortlet.java:151]: @W2jx Please update the commit message to let people know this class has a subclass so we need to keep these protected.

**Review by dantewang — APPROVED** [review]
@W2jx Code approved, please update commit message.

File: modules/dxp/apps/portal-reports-engine-console/portal-reports-engine-console-web/src/main/java/com/liferay/portal/reports/engine/console/web/internal/admin/portlet/AdminPortlet.java (hunk 1 above)
Comment (W2jx) [reply modules/dxp/apps/portal-reports-engine-console/portal-reports-engine-console-web/src/main/java/com/liferay/portal/reports/engine/console/web/internal/admin/portlet/AdminPortlet.java:151]: I will fix it.

**Comment by W2jx** [comment]
Force-push just to update commit message.

**Comment by W2jx** [comment]
ci:test:sf

**Comment by zhangyan0701** [comment]
ci:test:commerce

**Comment by zhangyan0701** [comment]
No regressions found.

**Comment by dantewang** [comment]
@shuyangzhou @tinatian All tests passed and changes are direct replacements. Asked Commerce team for a final check and forward.

**Comment by tinatian** [comment]
ci:forward

**Pull request closed without merging**